testing/execution.ts

```diff
@@ -87,10 +88,31 @@ export async function executeFormulaOrSyncFromCLI({
       : newMockSyncExecutionContext();
     executionContext.sync.dynamicUrl = dynamicUrl || undefined;
 
-    const result = vm
-      ? await executeFormulaOrSyncWithRawParamsInVM({formulaName, params, manifestPath, executionContext})
-      : await executeFormulaOrSyncWithRawParams({formulaName, params, manifest, executionContext});
-    print(result);
+    const syncFormula = helper.tryFindSyncFormula(manifest, formulaName);
+    if (syncFormula) {
+      const result = [];
+      let iterations = 1;
+      do {
+        if (iterations > MaxSyncIterations) {
```

Comment:
I meant to allow you to override this with a CLI flag, we should do that at some point.

Reply:
yeah, I found it was not currently used so just remove it for simplicity. we can always add this if needed.